src/ape/contracts/base.py

```diff
@@ -1267,6 +1267,23 @@ def deploy(self, *args, publish: bool = False, **kwargs) -> ContractInstance:
 
         return instance
 
+    def declare(self, contract: "ContractContainer", *args, **kwargs) -> ReceiptAPI:
```

Comment:
`self` is `ContractContainer`?
```suggestion
    def declare(self, *args, **kwargs) -> ReceiptAPI:
```

Reply:
whoops